packages/tldraw/src/lib/shapes/line/toolStates/Pointing.ts

```diff
@@ -106,15 +114,16 @@ export class Pointing extends StateNode {
 
 		if (this.editor.inputs.isDragging) {
 			const handles = this.editor.getShapeHandles(this.shape)
+			console
 			if (!handles) {
 				if (this.markId) this.editor.bailToMark(this.markId)
 				throw Error('No handles found')
 			}
-
+			const lastHandle = last(handles)!
 			this.editor.setCurrentTool('select.dragging_handle', {
 				shape: this.shape,
 				isCreating: true,
-				handle: last(handles)!,
+				handle: { ...lastHandle, x: lastHandle.x - 0.1, y: lastHandle.y - 0.1 },
```

Comment:
This looks weird. The end handle is initially set to the pos 0.1, 0.1 . When shift clicking, we update the position of the end handle to a new point. But, when dragging we actually need to shave the offset off somehow. The branch was failing Line Tool tests and any snapshot tests that drew lines without this change. This was what I came up with, but there may be a better way.